assets/_scss/components/_accordions.scss

```diff
@@ -1,41 +1,39 @@
 .usa-accordion {
+
   ul {
-    border: 1px solid #757575;
-    border-radius: 5px;
     color: $color-base;
     margin: 0;
     overflow: hidden;
     padding: 0;
 
     li {
-      background-color: #eeeeee;
-      border-bottom: 1px solid #757575;
+      background-color: $color-gray-lightest;
       font-family: $font-sans;
       list-style: none;
-      margin: 0;
-
-      &:last-child {
-        border-bottom: 0;
-      }
+      margin-bottom: 6px;
     }
 
     button[aria-expanded=false] {
-      background: url('../img/arrow-right.png') no-repeat;
-      background-position: 15px;
-      border-bottom: 0;
+      background: url('../img/plus.png') no-repeat;
+      background-position: 96%;
+      background-size: 13px;
     }
 
     button {
-      background: url('../img/arrow-down.png') no-repeat;
-      background-position: 15px;
-      border-bottom: 1px solid #757575;
+      background: url('../img/minus.png') no-repeat;
+      background-position: 96%;
+      background-size: 13px;
       color: $color-base;
       cursor: pointer;
       display: inline-block;
       font-family: $font-sans;
       margin: 0;
-      padding: 10px 15px 10px 35px;
+      padding: 15px 30px;
```

Comment:
I think these should be rem, or em units. Is there a specific reason they're px here?